src/templates/app/user/security/account_closure.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import Loading from '../../../_common/components/loading.jsx';
+import { SeparatorLine } from '../../../_common/components/separator_line.jsx';
+import { FormRow, SubmitButton } from '../../../_common/components/forms.jsx';
+
+const AccountClosure = () => (
+    <React.Fragment>
+        <div id='msg_main' className='center-text gr-gutter gr-padding-10 invisible'>
+            <h1>{it.L('Account closure confirmed')}</h1>
+            <p className='notice-msg'>{it.L(`Accounts closed successfully. A confirmation email will be sent to your email.[_1]This page will redirect to the ${it.website_name} homepage after 10 seconds.`, '<br />')}</p>
```

Comment:
We shouldn't include `it.website_name` as part of the localized string.
Should be:
```<p className='notice-msg'>{it.L('Accounts closed successfully. A confirmation email will be sent to your email.[_1]This page will redirect to the [_2] homepage after 10 seconds.' , '<br />', it.website_name)}</p>```